files/default/test-neutron-ha-tool.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+import collections
+ha_tool = __import__("neutron-ha-tool")
```

Comment:
That looks like a pretty unconventional way of doing the import.  Isn't it possible to do it with a normal `import`?